beets/test/fixtures.py

```diff
@@ -0,0 +1,92 @@
+# This file is part of beets.
+# Copyright 2024, Arav K.
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+
+"""
+This module provides `pytest`-based fixtures for testing Beets.
+"""
+
+from pathlib import Path
+from typing import Iterator
+
+import pytest
+from confuse import Configuration
+
+import beets
+from beets.library import Library
+
+
+@pytest.fixture(scope="session")
+def resource_dir() -> Path:
+    """
+    A fixture returning the resource directory for tests.
+    """
+
+    return Path(__file__).parents[2] / "test" / "rsrc"
+
+
+@pytest.fixture(scope="function")
+def config(
+    tmp_path: Path,
+    monkeypatch: pytest.MonkeyPatch,
+) -> Iterator[Configuration]:
+    """
```

Comment:
Yep, I usually follow that convention for function descriptions, but I wasn't sure whether it made sense in the case of fixtures (since from a user perspective, they're really some kind of automatically generated _value_, not a function).  I'm happy to switch to something like "Prepare configuration for the library" instead.

(P.S. I personally disagree with the stated syntax convention for one-liners, I like keeping the quotes on their own lines.  But if this is a popular convention then I'm happy to adjust these function descriptions accordingly.)

Reply:
Tbh ideally I'd see the docstrings removed from here - you've done a good job naming the fixtures and the implementation is simple - I don't see any reason for the docstring. 

In such cases I find docstrings counter-productive - they just occupy space and add noise 🤷🏼

Reply:
Actually, I want to add a bit more (non-obvious) information in these docstrings.  I can't expect every user to read and understand the implementation, even though it is simple.  I'll add some shortly -- if you still think it's not necessary, we can remove it all.

Reply:
I see, it's good stuff!